proposals/fleet-desktop-token-rotation.md

```diff
@@ -0,0 +1,64 @@
+# Token Rotation for Fleet Desktop
+
+This file is based on the original proposal described at [#6348](https://github.com/fleetdm/fleet/issues/6348) modified based on a few lessons we learned and the new communication channel between Orbit and the Fleet server introduced in https://github.com/fleetdm/fleet/issues/6851
+
+**Compatibility**
+
+|                | Fleet Desktop < v1.0.0            | Fleet Destkop >= v1.0.0 |
+| -------------- | --------------------------------- | ----------------------- |
+| Server < 4.21  | OK/Rotation disabled              | OK/Rotation disabled    |
+| Server >= 4.21 | Fleet Destkop breaks after 1 hour | OK/Rotation enabled     |
+
+
+## Fleet Server
+
+1. Add a new endpoint `POST /orbit/<orbit_node_key>/device_token` to create/update device tokens
```

Comment:
We could use what was built in [#6851](https://github.com/fleetdm/fleet/issues/6851).

```
// orbit authenticated endpoints
oe := newOrbitAuthenticatedEndpointer(svc, logger, opts, r, apiVersions...)
oe.POST("/api/_version_/fleet/orbit/flags", getOrbitFlagsEndpoint, orbitGetConfigRequest{})
```
So, something like
```
POST /api/_version_/fleet/orbit/desktop_token
```
(orbit node key auth is handled by `newOrbitAuthenticatedEndpointer`)

Reply:
makes sense, I will update 👍